malariagen_data/util.py

```diff
@@ -140,19 +138,15 @@ def da_from_zarr(z, inline_array, chunks="auto"):
 def dask_compress_dataset(ds, indexer, dim):
     """Temporary workaround for memory issues when attempting to
     index an xarray dataset with a Boolean array.
-
     See also: https://github.com/pydata/xarray/issues/5054
-
     Parameters
     ----------
     ds : xarray.Dataset
     indexer : str
     dim : str
-
     Returns
     -------
     xarray.Dataset
-
     """
```

Comment:
Did you mean to remove this whitespace?

Reply:
No, I think it might have been my formatter. I have added them back in now.